src/OpenTelemetry/Metrics/PeriodicExporterOptions.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="PeriodicExporterOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+public class PeriodicExporterOptions
```

Comment:
This might be misleading, it is actual options for reader, not exporter?

Reply:
Agreed the name is not quite right here. It was in part inspired by this section of the spec titled [Periodic exporting MetricReader](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#periodic-exporting-metricreader).

Few ideas:
1. Rename to `PeriodicExportingMetricReaderOptions`
2. Rename to `PeriodicMetricReaderOptions`
3. Do away with the options class and just keep the interval as a top-level property of the console options.

I think I like one of the rename options because the class kind of mimics what we have in tracing plus we should probably expand these options in the future with `ExportTimeoutMilliseconds`.

Do folks like any of these options?

Reply:
For now, we opted to rename `PeriodExporterOptions` to `PeriodicExportingMetricReaderOptions`.